js/components/operatorSignIn/list.tsx

```diff
@@ -0,0 +1,62 @@
+import { back } from "../../browser";
+import { findEmployeeByBadge, useEmployees } from "../../hooks/useEmployees";
+import { useSignins } from "../../hooks/useSignIns";
+import { fullName } from "../../models/employee";
+import { DateTime } from "luxon";
+import { ReactElement } from "react";
+
+export const List = ({ line }: { line: string }): ReactElement => {
+  const employees = useEmployees();
+  const signIns = useSignins(line);
+
+  if (signIns.status === "loading" || employees.status === "loading") {
+    return <>Loading...</>;
+  } else if (signIns.status === "error") {
+    return <>Error retrieving signins: {signIns.error}</>;
+  } else if (employees.status === "error") {
+    return <>Error retrieving employees: {employees.error}</>;
+  }
```

Comment:
**suggestion (non-blocking):** I don't have an immediate idea of how to implement this, but it feels like this could be factored out into some sort of reusable "loading or fetch error" component that could be used for various places where the application needs to wait for API responses.

Reply:
Yeah, agree. I'll file a task- want to get this merged in the immediate future.

Reply:
Sounds good!